user_guide/admin_capabilities.rst

```diff
@@ -0,0 +1,461 @@
+.. _admin-capabilities:
+
+Capabilities List
+=================
+
+The *"Capabilities"* make an important part of the access and permission settings defined in the system, as described in 
+the :ref:`Access and Permissions Basics guide <user-guide-user-management-permissions-basic>`. 
+
+Depending on their specific roles, users will have access to different functionalities. For example, some users may not 
+be able to assign tags, share grids, or create new users.
+
+For each role and capability pair there are only two options:
+
+For each role and capability pair there are only two options:
+
+- **None**: users with the role won’t be able to use the functionality.
+- ***System***: users with the role will be able to use the functionality for all the records created within their 
+  OroCRM instance they’ve logged in into.
+
+  
+.. image:: ./img/roles/01.png   
+  
+This guide describes the set of capabilities available in the alphabetical order and provides examples of several 
+default capability settings for some common roles.
+
+
+
+.. _admin-capabilities-acc:
+
+Abandoned Cart Campaign manipulations
+-------------------------------------
+
+If this capability is set to *"System"*, users with the role can use the Abandoned Cart Campaign functionality to 
+generate automatic mailings to customers who have unfinished their purchase in your Magento store in order to convince 
+them to complete the checkout and place an order.  You can find more details about the functionality in the 
+:ref:`Magento Abandoned Cart Campaigns guide <user-guide-acc>`.
+
+This capability is useful for marketing representatives and other employees working with email campaigns for customers 
+of Magento-based stores, e.g. for a general Magento store manager. The functionality may be reached from the 
+*Marketing* menu:
+
+
+.. important::
+
+    To use the functionality, you also need to :ref:`set up the Abandoned Cart extension <user-guide-acc-ext>`.
+
+    
+.. _admin-capabilities-data-audit:    
+    
+Data Audit
+----------
+
+If this capability is set to *"System"*, users with the role can see full history of changes made to any record of an 
+:ref:`auditable entity <user-guide-entity-management-create-other>` and an out-of-the-box report of all such actions.
+
+
+The report is available from the "*System*" menu and described in more details in the 
+:ref:`Data Audit <user-guide-data-audit>` guide.
+
+The link to the history of changes of a specific record is available in the top right corner of the record’s
+:ref:`View page <user-guide-ui-components-view-pages>`.
+
+|
+
+.. image:: ./img/data_management/view/view_history.png
+
+|
+
+This capability is usually used by system administrators and may be used by managers in order to see how and by whom 
+details of different records were changed.
```

Comment:
Edit: This capability is usually used by system administrators, but may also be used by managers in order to see how details of different records were changed, as well as by whom.